Detour/Source/DetourNavMesh.cpp

```diff
@@ -616,12 +616,102 @@ void dtNavMesh::baseOffMeshLinks(dtMeshTile* tile)
 	}
 }
 
+void dtNavMesh::closestPointOnDetailedContour(const dtMeshTile* tile, const dtPoly* poly, const float* pos, float* closest) const
```

Comment:
There's already `closestPointOnPolyBoundary` so this should be `closestPointOnDetailBoundary` for consistency.

Reply:
Should the flag be `DT_DETAIL_EDGE_BOUNDARY` too?

Reply:
Yes. Contours are the specific outlines before triangulation.